ansible/configs/controller-migration-casc/README.adoc

```diff
@@ -0,0 +1,50 @@
+= ansible-automation-platform config
+
+Author: Eric Lavarde <elavarde@redhat.com>
+Owner: Eric Lavarde <elavarde@redhat.com>
+Alternative Contacts: Goetz Rieger <grieger@redhat.com>
+
+== Overview
+
+This config is meant to be generic and reusable by others, but has been primarily developed
+to cover the needs of the Summit labs (formerly known as):
+- Getting Started with Ansible Tower
+- Ansible Tower Advanced Lab.
+
+== Setup
+
+Copy the `sample_vars.yml` to a place outside of the Git repo (for security reasons)
+and adapt the variables to your needs. You can also add and overwrite variables
+present in the link:env_vars.yml[env_vars.yml file].
+
+TIP: you'll need to adapt _once_ all variables marked with `CHANGEME` to adapt them to your own person.
+The variables marked with `CHANGEME-ec2` need to be adapted once for each new AWS-sandbox deployment (as long as you keep the same sandbox, they don't need to change).
+The GUID only needs to be different if you deploy the same environment multiple times in the same sandbox.
+
+== Running Ansible Playbook
+
+From the `agnosticd/ansible` directory run:
+
+[source,bash]
+----
+ansible-playbook main.yml  \
+      -e @.../my_adapted_sample_vars_with_my_secrets.yml \
+----
```

Comment:
can you add recommendation about what Execution Environment to use? Like ee-multicloud:latest for example?

Reply:
These lines are running the `ansible-playbook` command, which doesn't need any EE to be specified. The Ansible Controller provisioning jobs during the development tests are running behind the EE called `quay.io/agnosticd/ee-multicloud:v0.1.1`, but I don't see how to link it with the `ansible-playbook` command shown into the `README.md` file...

Reply:
TIP added regarding that the playbook should work with `>= v0.1.1.`